x-pack/plugins/security_solution/public/timelines/components/timeline/data_providers/provider.tsx

```diff
@@ -32,7 +37,8 @@ export const Provider = React.memo<OwnProps>(({ dataProvider }) => {
       toggleExcludedProvider={noop}
       toggleEnabledProvider={noop}
       toggleTypeProvider={noop}
-      val={dataProvider.queryMatch.displayValue || dataProvider.queryMatch.value}
```

Comment:
This (original, pre-PR) code appears to be incorrect, because a `displayValue` may be very different than the value that should be used to execute the query

Reply:
I agree, but was concerned this was done as some sort of bug fix. Will set it to enfoce always being value